ci-operator/step-registry/acm/interop-p2p/submariner-diag/acm-interop-p2p-submariner-diag-commands.sh

```
@@ -0,0 +1,246 @@
+
+# GatherSubmariner — subctl gather for deep diagnostics; archives tarball to diagDir.
+GatherSubmariner() {
+    typeset kc="${1:?}"; (($#)) && shift
+    typeset name="${1:?}"; (($#)) && shift
+
+    typeset gatherWorkDir
+    gatherWorkDir="$(mktemp -d /tmp/subctl-gather-XXXXXX)"
+    mkdir -p "${diagDir}/gather"
+
+    (
+        cd "${gatherWorkDir}"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)